0001-chore-add-debian-package-to-workflow.patch

```diff
@@ -0,0 +1,66 @@
+From 73a7363ea1bbb8622ded0507dc12ba0483523894 Mon Sep 17 00:00:00 2001
+From: Denis Benato <benato.denis96@gmail.com>
+Date: Sun, 9 Mar 2025 16:11:05 +0100
+Subject: [PATCH 1/2] chore: add debian package to workflow
```

Comment:
We shouldn't be adding these patches to source control here.

Reply:
Damn. My bad.